posts/2020-10-27-microprofile40-jakartaee9-beta-200012.adoc

```diff
@@ -44,9 +44,7 @@ Open Liberty 20.0.0.12-beta provides another two feature updates for MicroProfil
 * <<conf, MicroProfile Config 2.0>>
 * <<jwt, MicroProfile JWT 1.2>>
 
-Other MicroProfile 4.0 features were part of earlier betas: MicroProfile OpenAPI 2.0 in link:{url-prefix}/blog/2020/09/29/microprofile-openapi-beta-200011.html#openAPI[20.0.0.11-beta]; MicroProfile Fault Tolerance 3.0, MicroProfile Metrics 3.0, MicroProfile Health 3.0, and MicroProfile OpenTracing 2.0 in link:{url-prefix}/blog/2020/09/02/jakarta-microprofile-beta-200010.html#allbeta[20.0.0.10-beta].
-
-While we'd hoped to also include MicroProfile Rest Client 2.0 (also part of MicroProfile 4.0), we didn't quite make it this time so it will make its debut in a future beta.
+Other MicroProfile 4.0 features were part of earlier betas; with MicroProfile Fault Tolerance 3.0, MicroProfile Metrics 3.0, MicroProfile Health 3.0, and MicroProfile OpenTracing 2.0 in the Open Liberty link:{url-prefix}/blog/2020/09/02/jakarta-microprofile-beta-200010.html#allbeta[20.0.0.10-beta], followed by MicroProfile OpenAPI 2.0 in link:{url-prefix}/blog/2020/09/29/microprofile-openapi-beta-200011.html#openAPI[20.0.0.11-beta]. While we'd hoped to also include MicroProfile Rest Client 2.0 (also part of MicroProfile 4.0), we didn't quite make it this time so it will make its debut in a future beta.
```

Comment:
I think we can add:
"However, if you just can't wait until then, you can always give the feature a try in one of our [nightly builds](https://openliberty.io/downloads/#development_builds).

Reply:
Added, will appear in next pr to avoid spamming you for approval.

Reply:
sounds good; when you do that, don't forget about the comment below as well:
"Also, I would expect to see either 2020-10-23-kerberos-grafana-200011.adoc.adoc getting renamed to 2020-10-23-kerberos-grafana-200011.adoc, or 2020-10-23-kerberos-grafana-200011.adoc.adoc being deleted."